src/js/pages/More/Donate.jsx

```diff
@@ -380,16 +377,16 @@ class Donate extends Component {
                           </ContributeGridSection>
                         </ContributeGridWrapper>
                         <PaymentWrapper joining={joining}>
-                          <PaymentCenteredWrapper>
```

Comment:
I'd also like to leave this page unchanged this close to the election. This code shouldn't be reachable on our live site, but there are too many moving parts to make this change right now.